tools/refguide_check.py

```diff
@@ -55,13 +57,23 @@ def get_all_dict(module):
         except ValueError:
             pass
 
+    # FIXME: shouldn't modules be in the refguide, actually? if they're in __all__?
     # somehow some modules survive the first iteration (?)
     for _ in range(2):
         for name in all:
             if inspect.ismodule(getattr(module, name)):
                 all.remove(name)
 
-    return all
+    deprecated = []
+    for name in all:
+        f = getattr(module, name)
+        if name in ['info', 'who']: # These print to stdout when called
```

Comment:
Is it feasible to redirect to /dev/null generically? Exceptions are annoying. I can remove some of them by using inspect to intentionally supply the wrong numbers of arguments, which doesn't stop the deprecation warning from triggering but will dissuade non-varadic functions from doing anything. Ooh, a bogus keyword argument should generically prevent successful calling, right?